code/modules/cargo/expressconsole.dm

```diff
@@ -26,6 +33,27 @@
 	// This also sets this on the circuit board
 	var/obj/item/circuitboard/computer/cargo/board = circuit
 	board.emagged = TRUE
+	oh_shit_im_sorry()
+	
+/obj/machinery/computer/cargo/express/proc/oh_shit_im_sorry()

```

Comment:
choose a reasonable name

Reply:
![it_gets_bigger_when_i_merge_it](https://file.house/QlK-.png)

Walking into this not once, but three times disqualifies you, sorry

Reply:
you really dont know me if you don't realize that I love to engage in pointless argument

Reply:
This is how you do a proper meme name

https://github.com/tgstation/tgstation/blob/master/code/modules/power/tesla/energy_ball.dm#L106-L107